AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_Transformer_AMX_bfloat16_MixedPrecision/IntelTensorFlow_Transformer_AMX_bfloat16_MixedPrecision.ipynb

```diff
@@ -31,10 +31,18 @@
   },
```

Comment:
remove edits from this file

Reply:
I see that some of the edits are needed. I did not test it though.
I just removed the output lines